.travis.yml

```diff
@@ -16,7 +16,7 @@ matrix:
   - scala: 2.11.2
     script:
       - ./scripts/wait-for-it.sh 127.0.0.1:9042 -- echo "Cassandra/ScyllaDB is up"
-      - ./sbt test
+      - ./sbt "test-only -- -l Cassandra"
```

Comment:
Why do you think this is required?

Having sub-sets of test labeled is very nice (esp. to filter out those big ones, that has external dependencies), but for this CI profile, it seems that it most probably would make sense to execute all of them. 

This has additional benefit of being a very simple `./sbt test` which is quite standard.

Reply:
The tests-cases labeled as "Cassandra" requires Apache Cassandra, not available in the CI (because we use Scylla)

If we could use two different database images with Travis, we could use the Cassandra one to pass these tests (but I think it should be done in a separated PR)

Reply:
Thank you for explanation! If it excludes a Cassandra ones - that sounds good!

For now, we do not want to support 2 DB, so the solution looks good.

Reply:
:+1: You can find that changes (excluding the Cassandra tagged test cases) in this commit https://github.com/src-d/gemini/pull/11/commits/0c661c6b9952e398892443b732976149d4f2b48a